### PR TITLE
Cloudsql

### PR DESCRIPTION
**Pull request by seslattery**

### CODE REVIEW

File: pachyderm/templates/pachd/deployment.yaml [hunk 1]
```diff
@@ -38,6 +38,32 @@ spec:
       - command:
         - /pachd
         env:
+        {{- if .Values.postgresql.host }}
```
Comment (chainlink) [inline pachyderm/templates/pachd/deployment.yaml:41]: these need to be updates to `.Values.pachd.`

File: pachyderm/templates/pachd/storage-secret.yaml [hunk 2]
```diff
@@ -43,3 +43,6 @@ data:
   MICROSOFT_ID: {{ .Values.pachd.storage.microsoft.id | toString | b64enc | quote }}
   MICROSOFT_SECRET: {{ .Values.pachd.storage.microsoft.secret | toString | b64enc | quote }}
   {{- end }}
+  {{- if  .Values.postgresql.password }}
```
Comment (chainlink) [inline pachyderm/templates/pachd/storage-secret.yaml:46]: same here

File: pachyderm/templates/postgresql/storageclass.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{{- /*
```
Comment (chainlink) [inline pachyderm/templates/postgresql/storageclass.yaml:1]: this file was removed

**Review by chainlink — CHANGES_REQUESTED** [review]

File: pachyderm/templates/postgresql/storageclass.yaml (hunk 3 above)
Comment (seslattery) [reply pachyderm/templates/postgresql/storageclass.yaml:1]: huh, wonder why that didn't show in the rebase.  Ok I'll kill it

**Review by chainlink — APPROVED** [review]

**Pull request merged**